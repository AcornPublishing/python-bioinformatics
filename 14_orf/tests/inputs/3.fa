>Rosalind_4620
GTGGATATGACCTCACCAGAATATCGGACATCGGCGCGGGCTTGATTAAACAGCAAGACT
GCATTAAAGGACGTTACCGGAGACCTACCAGGACAATAGATGCTCGCGGGACATGTTGGG
TGGCCTTAATTACGGGTAAGAGTGAGCCTAGGATTTGGATCCAAGAAGCCTTCTGAGAAT
AATCCAGCGGTCGTAAATTCCAGCCACCATGCGCATGAAGCATGCCTTCCGTATCAGTTT
TCAGGCAAACAATTGTGTGTGGGTCAACTAAACTGCTGGTGATCACCATGAATGTCCTGG
CGCTATGCTATTTTCTGCGGTTTAGAACGAATCGCCGCCCAGGAGGAGCGTCGGAAGTTG
CAATCGATACTTATTATTATCTGCACGCAACCGCCTGTCGATGGCTATTGGCGTAGTCTG
GGTATAGCTATACCCAGACTACGCCAATAGCCATCGCGCGCCGATCCATCGGAACCGGCT
AGGTCACAGGAGCGAGTAGAACGGTACCGAAGCTGTATTGCTCGCCGTGCTTTATAATAT
CGTTATCGGATGCTCGGATATCGGCTACATAGGATGCGGACTATACCAAGGCCCAGACAT
GAATCGTTGTAGATTTTACATACCCCCCGAAATGCCAACAGCAGAAGGCTACCCGCATGG
TGAAGGCCCTTCCACCTTATGAGTATACGAGTAAATGGCCAACGGCGTGGTAGCGACCGG
ATTGGGCCGCTCCCTTCTTGCCTAATCCTCTAATCTATGGCCTGCCTAGCGCCGAGGGTG
CCTGTGAGTTGATTCAATACGCTCTCAGCATTACATATAGCACCCTTTGAGGGTACTCGT
AGAAGACATTGGAC
