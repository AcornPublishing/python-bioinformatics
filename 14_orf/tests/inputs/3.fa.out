M
MACLAPRVPVS
MAIGVVWV
MANGVVATGLGRSLLA
MDRRAMAIGVVWV
MKHAFRISFQANNCVWVN
MLAGHVGWP
MLFSAV
MLGGLNYG
MLGYRLHRMRTIPRPRHESL
MLHAHGGWNLRPLDYSQKASWIQILGSLLPVIKATQHVPRASIVLVGLR
MLRAY
MNRCRFYIPPEMPTAEGYPHGEGPSTL
MNVLALCYFLRFRTNRRPGGASEVAIDTYYYLHATACRWLLA
MPSVSVFRQTIVCGSTKLLVITMNVLALCYFLRFRTNRRPGGASEVAIDTYYYLHATACRWLLA
MPTAEGYPHGEGPSTL
MRMKHAFRISFQANNCVWVN
MRMVAGIYDRWIILRRLLGSKS
MRTIPRPRHESL
MRVAFCCWHFGGYVKSTTIHVWALV
MSGPWYSPHPM
MSIRVNGQRRGSDRIGPLPSCLIL
MSREHLLSW
MSSTSTLKGCYM
MSWRYAIFCGLERIAAQEERRKLQSILIIICTQPPVDGYWRSLGIAIPRLRQ
MTSPEYRTSARA
MVAGIYDRWIILRRLLGSKS
MVITSSLVDPHTIVCLKTDTEGMLHAHGGWNLRPLDYSQKASWIQILGSLLPVIKATQHVPRASIVLVGLR
MVKALPPYEYTSKWPTAW
